src/tools/podToMD.pl

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env perl
+#*************************************************************************
+# Copyright (c) 2013 UChicago Argonne LLC, as Operator of Argonne
+#     National Laboratory.
+# SPDX-License-Identifier: EPICS
+# EPICS BASE is distributed subject to a Software License Agreement found
+# in file LICENSE that is included with this distribution.
+#*************************************************************************
+
+use strict;
+use warnings;
+
+# To find the EPICS::Getopts module used below we need to add our lib/perl to
+# the lib search path. If the script is running from the src/tools directory
+# before everything has been installed though, the search path must include
+# our source directory (i.e. $Bin), so we add both here.
+use FindBin qw($Bin);
+use lib ("$Bin/../../lib/perl", $Bin);
+
+use EPICS::Getopts;
+
+use Pod::Markdown;
+
+use Pod::Usage;
+
+=head1 NAME
+
+podToMD.pl - Convert EPICS .pod files to .md
+
+=head1 SYNOPSIS
+
+B<podToMD.pl> [B<-h>] [B<-o> file.md] file.pod
+
+=head1 DESCRIPTION
+
+Converts files from Perl's POD format into Markdown format.
+
+=head1 OPTIONS
+
+B<podToMD.pl> understands the following options:
+
+=over 4
+
+=item B<-h>
+
+Help, display this document as text.
+
+=item B<-o> file.md
+
+Name of the Markdown output file to be created.
+
+=back
+
+If no output filename is set, the file created will be named after the input
+file, removing any directory components in the path and replacing any file
+extension with .md.
+
+=cut
+
+our ($opt_o, $opt_h);
+
+sub HELP_MESSAGE {
+    pod2usage(-exitval => 2, -verbose => $opt_h);
+}
+
+HELP_MESSAGE() if !getopts('ho:') || $opt_h || @ARGV != 1;
+
+my $infile = shift @ARGV;
+
+my @inpath = split /\//, $infile;
+my $file = pop @inpath;
+
+if (!$opt_o) {
+    ($opt_o = $file) =~ s/\. \w+ $/.md/x;
+}
+
+open my $out, '>', $opt_o or
+    die "Can't create $opt_o: $!\n";
+
+$SIG{__DIE__} = sub {
+    die @_ if $^S;  # Ignore eval deaths
+    close $out;
+    unlink $opt_o;
+};
+
+my $podMD = Pod::Markdown->new();
+
+$podMD->perldoc_url_prefix('');
+# $podMD->perldoc_url_postfix('.md');
```

Comment:
Yes, that is true. Currently, the URLs that are broken are URLs to another file, with a fragment. So a URL like this are broken: `dbCommonRecord#Alarm-Fields`.

URLs pointing to just another file, or just another section are fine.

Reply:
@minijackson would probably solve this much quicker, but if he does not find the time, I can take this as a challenge to improve my Perl skills. I cannot guarantee quick (or any, for that matter) results though...

Reply:
I do *not* feel confident about my Perl skill, nor about maintaining an in-house fork of Pod::Markdown.

If possible, I'd like to leave this issue for later, as to not block this PR further.

If it's ok, a workaround might be to replace these links from this: `L<Alarm Fields|dbCommonRecord/Alarm Fields>` to this: `L<Alarm Fields|dbCommonRecord.md/Alarm Fields>`.

This will fix the links for Read the Docs, which if I'm not mistaken should be the default documentation entry point going forward.

Reply:
I am not confident in Perl either, but I looked at the code that Andrew pointed out, and I think I should be able to figure it out. The code that Andrew referred to is not meant to be an implementation of POD to MD conversion, but a rather short snippet to do the link mangling.
However, if I do not manage to solve this in a couple of days, I would be OK to leave this issue to be solved later so that the PR will not be delayed further. It would also be easier (for me at least) to fix this issue separately.
I am a bit hesitant to do the workaround with the links, at least yet. It would solve the issue for RTD, but break the current RRM builds. And, there would be quite many files to change. 
If you give me a couple of days to try this out, I will comment again by the end of the week if I managed.

Reply:
Thanks a lot @ttkorhonen! If you don't mind, I do prefer leaving this issue for another PR. Feel free to tag me on that one if you want.

Reply:
Fine with me! I think it is even better that way, makes things simpler. From my point of view, this PR would be good to go.

Reply:
@ttkorhonen I'd be happy to help if you get stuck. If you haven't already take a look at [this code](https://github.com/rwstauner/Pod-Markdown/blob/9866b1c8d3c6193e91417668ae2f8e74a94e8467/lib/Pod/Markdown.pm#L1221) and below it in the `Pod::Markdown` module which seems to be the part that formats links. Your new `EPICS::PodMarkdown` module would be derived from `Pod::Markdown` and provide its own implementation of one of the routines used there (which it can call using `$self->SUPER::...()` for other link types), similar to the `sub do_pod_link` in my `EPICS::PodHtml` module. Unfortunately every Pod processing module does things slightly differently so you'll have to discover exactly how to detect the link type and process the result appropriately.

Reply:
Thanks to a bit slower business before holidays I found some free time to figure out this issue - and in fact, I managed to find a solution. :-) I followed your instructions, @anjohnson , and derived a new module for Pod::Markdown which overrides one function of the parent module. 
I would still prefer to do that change in a follow-up PR, not to make this one more complicated than it already is.

Reply:
I would be grateful if this PR gets attention relatively soon, before I forget the solution I mentioned above... ;-)